core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/BlockCache.java

```diff
@@ -17,10 +17,30 @@
  */
 package org.apache.accumulo.core.file.blockfile.cache;
 
+import org.apache.accumulo.core.conf.AccumuloConfiguration;
+
 /**
  * Block cache interface.
  */
 public interface BlockCache {
+
+  /**
+   * Start the block cache
+   *
+   * @param conf
+   *          Accumulo configuration object
+   * @param maxSize
+   *          maximum size of the on-heap cache
+   * @param blockSize
+   *          size of the default RFile blocks
+   */
+  void start(AccumuloConfiguration conf, long maxSize, long blockSize) throws Exception;
```

Comment:
re: POJO -> I was going to just use the AccumuloConfiguration and the custom properties for this. I'm not sure how a single BlockCacheConfiguration class would work across different BlockCache impls that have different configuration parameters.